.envrc

```diff
@@ -0,0 +1,2 @@
+test -f .env && dotenv
```

Comment:
What's this file for?

Reply:
https://direnv.net/

It allows `.env` files to be used (by loading the contents into one's environment separately) without necessitating tools like foreman or libraries like `dotenv` that tie more tightly to `.env` files.

(It's really convenient for me but isn't necessary.)

Reply:
Ok, then I think this needs to be in `.gitignore`.